cli/src/cluster_query.rs

```diff
@@ -432,6 +432,13 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .takes_value(true)
                         .help("Start with the first signature older than this one"),
                 )
+                .arg(
+                    Arg::with_name("until")
+                        .long("until")
+                        .value_name("TRANSACTION_SIGNATURE")
```

Comment:
nit: do we have a sig validator?

Reply:
Looks like we do not! I won't add that here, but could be done separately

Reply:
kk. i'm indifferent. was more of a, "if it's there, may as well," thing